src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -279,9 +257,6 @@ public void stop() {
    */
   @Override
   public void open(final Collection<TopicPartition> partitions) {
-    this.taskOpenCount++;
```

Comment:
While this is definitely useful to keep track of, it was confusing for the tag. Easier to 'grep' on the KC instanceid and the taskid, then diff on how many different hashcodes are returned to see the open count